docs/quickstart.md

```diff
@@ -81,147 +36,6 @@ Run the following script if you want to destroy your local Armada cluster:
-
-# Install pulsar
-helm install pulsar apache/pulsar
-
-# Install Prometheus
-helm install kube-prometheus-stack prometheus-community/kube-prometheus-stack -f docs/quickstart/server-prometheus-values.yaml
-
-# Install Armada server
-helm install armada-server gresearch/armada -f ./docs/quickstart/server-values.yaml
-
-# Get server IP for executors
-SERVER_IP=$(kubectl get nodes quickstart-armada-server-worker -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
-```
-
-#### Executor deployments
-
-First executor:
-
-```bash
-kind create cluster --name quickstart-armada-executor-0 --config ./docs/quickstart/kind-config-executor.yaml
-
-# Set cluster as current context
-kind export kubeconfig --name=quickstart-armada-executor-0
-
-# Install Prometheus
-helm install kube-prometheus-stack prometheus-community/kube-prometheus-stack -f docs/quickstart/executor-prometheus-values.yaml
-
-# Install executor
-helm install armada-executor gresearch/armada-executor --set applicationConfig.apiConnection.armadaUrl="$SERVER_IP:30000" -f docs/quickstart/executor-values.yaml
-helm install armada-executor-cluster-monitoring gresearch/executor-cluster-monitoring -f docs/quickstart/executor-cluster-monitoring-values.yaml
-
-# Get executor IP for Grafana
-EXECUTOR_0_IP=$(kubectl get nodes quickstart-armada-executor-0-worker -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
-```
-
-Second executor:
-
-```bash
-kind create cluster --name quickstart-armada-executor-1 --config ./docs/quickstart/kind-config-executor.yaml
-
-# Set cluster as current context
-kind export kubeconfig --name=quickstart-armada-executor-1
-
-# Install Prometheus
-helm install kube-prometheus-stack prometheus-community/kube-prometheus-stack -f docs/quickstart/executor-prometheus-values.yaml
-
-# Install executor
-helm install armada-executor gresearch/armada-executor --set applicationConfig.apiConnection.armadaUrl="$SERVER_IP:30000" -f docs/quickstart/executor-values.yaml
-helm install armada-executor-cluster-monitoring gresearch/executor-cluster-monitoring -f docs/quickstart/executor-cluster-monitoring-values.yaml
-
-# Get executor IP for Grafana
-EXECUTOR_1_IP=$(kubectl get nodes quickstart-armada-executor-1-worker -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
-```
-
-#### Armada Lookout UI
-
-```bash
-# Set cluster as current context
-kind export kubeconfig --name=quickstart-armada-server
-
-# Install postgres
-helm install postgres bitnami/postgresql --set auth.postgresPassword=psw
-
-# Run database migration
-helm install lookout-migration gresearch/armada-lookout-migration -f docs/quickstart/lookout-values.yaml
-
-# Install Armada Lookout
-helm install lookout gresearch/armada-lookout -f docs/quickstart/lookout-values.yaml
-```
-
-You can view the UI by running the following:
-```bash
-kubectl port-forward svc/armada-lookout 8080:8080
-```
-You will be able to view Lookout at `http://localhost:8080`.
-
-#### Grafana configuration
```

Comment:
This block should also stay, even tho you get the same info from the install script

Reply:
Yea, think of the script just installing what you need to a k8 cluster.  

You still need to run these commands in a separate terminal in order to actually view the lookout UI.  

Same for Grafana/Prometheus